lib/test.js

```diff
@@ -106,8 +107,11 @@ Test.prototype.test = function (name, opts, cb) {
     });
 };
 
-Test.prototype.comment = function (msg) {
+Test.prototype.comment = function () {
     var that = this;
+    var keys = Object.keys(arguments)
+    var msg = keys.length > 1 ? format.apply(null, arguments) : arguments[keys[0]];
```

Comment:
Ah, interesting. We definitely want to avoid breaking changes.

Either way, I think perhaps the ES3 equivalent of `format.apply(null, Array.prototype.map.call(arguments, String))` might be the semantic we want here? What's the use case for supporting `format`'s behavior with non-strings?